src/hypothesis/strategies.py

```diff
@@ -33,6 +33,7 @@
 from hypothesis.utils.conventions import not_set
 from hypothesis.internal.reflection import proxies
 from hypothesis.searchstrategy.reprwrapper import ReprWrapperStrategy
+from hypothesis.searchstrategy.regex import Xeger
```

Comment:
So I suspect this is the line that’s causing the tests some grief. You import from `regex.py` here… which in turn imports from `strategies.py`… which brings us back to this line. The circular import is throwing everything out of whack. I don’t have time to review properly right now – and this is only a guess, the test output is very opaque – but maybe that’s worth a look?